telegram/replymarkup.py

```diff
@@ -31,3 +30,11 @@ class ReplyMarkup(TelegramObject):
     """
 
     __slots__ = ()
+
+    @staticmethod
+    def _check_keyboard_type(keyboard: object) -> bool:
+        """Checks if the keyboard provided is of the correct type - A list of lists."""
+        for row in keyboard:  # type: ignore[attr-defined]
```

Comment:
this will throw an exception if `keyboard` is not iterable. please add a check that `keyboard` is a list and also add a corresponding test case :)